library/core/src/intrinsics/simd.rs

```diff
@@ -612,6 +612,20 @@ extern "rust-intrinsic" {
     #[rustc_nounwind]
     pub fn simd_fma<T>(x: T, y: T, z: T) -> T;
 
+    /// Computes `(x*y) + z` for each element, non-deterministically executing either
+    /// a fused multiply-add or two operations with rounding of the intermediate result.
```

Comment:
Do we guarantee that the choice made is consistent across all lanes? Or could it happen that some lanes get fused and others not?

I assume this can happen because we want to allow the backend to split a big SIMD op into multiple smaller SIMD ops, and then some may be fused and some may not.